install/0000_30_machine-api-operator_09_rbac.yaml

```diff
@@ -11,6 +11,22 @@ metadata:
   name: machine-api-controllers
   namespace: openshift-machine-api
 
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
this is superseding https://github.com/openshift/machine-api-operator/pull/473/files#diff-2043d8125934ffff48ca0cb4d630543bR376 can we drop it here?

Reply:
Fixed.